sql/item_mods_pet.sql

```diff
@@ -135,7 +135,7 @@ INSERT INTO `item_mods_pet` VALUES (14958,63,5,0);      -- Beast Bazubands Pet:
 INSERT INTO `item_mods_pet` VALUES (15031,384,300,3);    -- Pantin Dastanas Automaton: Haste +3%
 INSERT INTO `item_mods_pet` VALUES (15032,384,300,3);    -- Pantin Dastanas +1 Automaton: Haste +3%
 INSERT INTO `item_mods_pet` VALUES (15101,165,3,1);     -- Summoner's Dblt. Avatar: Critical Hit Rate +3%
-INSERT INTO `item_mods_pet` VALUES (15116,25,7,1);      -- Summoner's Brcr. Avatar: Enhances accuracy +7 (?)
+INSERT INTO `item_mods_pet` VALUES (15116,25,14,1);      -- Summoner's Brcr. Avatar: Enhances accuracy +14
```

Comment:
Same with this one. Unsure where you pulled this value.

Reply:
https://kegsay.livejournal.com/4471.html

Couldn't find an updated value for this.

Reply:
https://www.bg-wiki.com/bg/Summoner%27s_Brcr. lists it as +7

Reply:
touche